ushahidi/platform-client
# Feature/1119 hdx button data export WIP

This pull request makes the following changes:
- Adds a warning-text to the user if no api-key is saved
- Adds a button to hdx
- Adds a route, controller and test-file for hdx

*Waiting on*
User api-key-settings-UI (can be tested through adjusting the db)
2807-user-settings in the api is needed


Testing checklist:
### QA-test checklist
Assuming HDX-flag is on
- Go to settings-data-export without any api-key saved
- [ ] 3 buttons should be visible, the last one greyed out
- [ ] An alert with a link to api-key-config should be visible
- Configure api-key and go back to settings-data-export
- [ ] 3 buttons should be visible, all of them active
- [ ] Click on the button that says "Assign tags and attributes"
- [ ] You should be directed to the tags and attributes page (doesnt need to contain anything at this stage, but the final result should look like this: https://xd.adobe.com/view/2690f082-d88d-4788-5db7-b04c9474a404-50a1/screen/9a17d59a-cdfd-4ed4-aec5-573cdfeadb97/Data-export-HDX-single-survey)



Fixes ushahidi/platform# .

Ping @ushahidi/platform
